cmake/SearchForStuff.cmake

```diff
@@ -164,9 +164,12 @@ if (PKG_CONFIG_FOUND)
   # Find Simbody
   set(SimTK_INSTALL_DIR ${SimTK_INSTALL_PREFIX})
   #list(APPEND CMAKE_MODULE_PATH ${SimTK_INSTALL_PREFIX}/share/cmake)
-  find_package(Simbody)
-  if (Simbody_FOUND)
-    message (STATUS "Looking for Simbody - found")
+  find_package(Simbody CONFIG)
```

Comment:
Unless someone is shipping a `FindSimbody.cmake` and putting it in the `CMAKE_MODULE_PATH`, I guess there is no reason to explicitly ask for `CONFIG`,right?
```suggestion
  find_package(Simbody)
```

Reply:
This is how it done for vcpkg.
I think it not find it without config.

Reply:
I will try.

Reply:
The vcpkg give the correct parameters to find it with config.
I will try without.

Reply:
ok. I already trying all your suggestion. Something is not working, and I am not sure what.

Reply:
done.